src/components/layout/_layout.scss

```diff
@@ -59,7 +59,7 @@ $includeHtml: false !default;
         flex-wrap: wrap;
         padding: gutter(1 / 2);
         max-width: $layoutMaxWidth;
-        background-color: $graySecondaryLightest;
+        background-color: $graySecondaryUltraLight;
```

Comment:
We would need to update background color for body in brainly-frontend as well. I will do it when I bump SG.